FRW/Services/Manager/UserManager.swift

```diff
@@ -662,7 +666,12 @@ extension UserManager {
 // MARK: - Switch Account
 
 extension UserManager {
+    @MainActor
     func switchAccount(withUID uid: String) async throws {
+        forceHideLoginScreen = true
+        defer {
+            forceHideLoginScreen = false
```

Comment:
if the func those error during account switching, the flag will still set to true

Reply:
not sure what you mean.  afaik defer still gets called even if an error is thrown?

Reply:
Are you sure the defer will be called even error is thrown ?